config/routes.rb

```diff
@@ -5,6 +5,15 @@
 require 'sidekiq/web' if ENV.fetch('HYRAX_ACTIVE_JOB_QUEUE', 'sidekiq') == 'sidekiq'
 
 Rails.application.routes.draw do # rubocop:disable Metrics/BlockLength
+  Hyrax.config.curation_concerns.each do |resource|
+    next if resource.to_s.match?(/.+Resource\z/)
+
+    resource = resource.to_s.underscore
+
+    get "/concern/#{resource}s/:id/edit", to: redirect("/concern/#{resource}_resources/%{id}/edit")
```

Comment:
It's possible that we don't need to use a redirect but could instead specify the controller/action pair.  Since this route is declared earlier than Hyrax, this route will take precedence.

Why avoid using a redirect?  It saves at least one HTTP request.

Reply:
cool i'm not sure what that means but i'll do some research